src/main/python/afp_cli/cliv2.py

```diff
@@ -71,8 +74,9 @@ def unprotected_main():
         log.DEBUG = True
     debug(arguments)
 
-    # parse the subcommand, only one will be active
-    subcommand = [s for s in SUBCOMMANDS if arguments[s]][0]
+    # parse the subcommand, use SIMPLE mode if no subcommand
+    subcommand = [s for s in SUBCOMMANDS if arguments[s]]
+    subcommand = subcommand[0] if subcommand else SIMPLE
```

Comment:
Is it really good Python to change the type of the variable from list to scalar?